presto-spi/src/main/java/com/facebook/presto/spi/InMemoryRecordSet.java

```diff
@@ -251,20 +253,6 @@ public InMemoryRecordSet build()
         }
     }
 
-    private static void checkArgument(boolean test, String message, Object... args)
```

Comment:
I'm not sure it's a good idea to expose such basic things as public functions in SPI.